autoPyTorch/pipeline/components/setup/network_backbone/utils.py

```diff
@@ -92,15 +92,35 @@ def backward(ctx: typing.Any,
 shake_drop = ShakeDropFunction.apply
 
 
-def shake_get_alpha_beta(is_training: bool, is_cuda: bool
-                         ) -> typing.Tuple[torch.tensor, torch.tensor]:
+def shake_get_alpha_beta(
+        is_training: bool,
+        is_cuda: bool,
+        method: str
+) -> typing.Tuple[torch.tensor, torch.tensor]:
+    """
+    The methods used in this function have been introduced in 'ShakeShake Regularisation'
+    https://arxiv.org/abs/1705.07485. The names have been taken from the paper as well.
+    """
     if not is_training:
         result = (torch.FloatTensor([0.5]), torch.FloatTensor([0.5]))
         return result if not is_cuda else (result[0].cuda(), result[1].cuda())
 
     # TODO implement other update methods
-    alpha = torch.rand(1)
-    beta = torch.rand(1)
+    if method == 'even-even':
```

Comment:
Wait. I checked [the paper](https://arxiv.org/pdf/1705.07485.pdf), but `shake-even` in the paper is equivalent to `even-even` here and there is nothing about `shake-even` in the paper.

Reply:
shake-even means sampling alpha and keeping beta as 0.5. 
even even is keeping both alpha and beta at 0.5

Reply:
`shake-even` means that you shake `alpha` and you keep `beta` as 0.5.
From the paper:
```
"Shake" means that all scaling coefficients are overwritten
with new random numbers before the pass. "Even" means that all scaling coefficients are set to
0.5 before the pass.
```

Reply:
so if the method is not `even-even` it will shake alpha and beta will be 0.5, if my understanding is correct. `even-even` should be 0.5 both.